internal/control-panel/registry.lua

```diff
@@ -77,6 +77,9 @@ COMMANDS_BY_IDX = {
         params={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/stuck-instruments', ']'}},
     {command='fix/stuck-worship', group='bugfix', mode='repeat', default=true,
         params={'--time', '1', '--timeUnits', 'days', '--command', '[', 'fix/stuck-worship', ']'}},
+    {command='fix/noexert-exhaustion', group='bugfix', mode='repeat', default=true,
+        desc='Prevent NOEXERT units from becoming Tired',
```

Comment:
```suggestion
```

Reply:
control panel picks up the short description from the rst help file by default, which is fine here.